table_schema.go

```diff
@@ -61,7 +61,8 @@ func fetchTableSchemas(ctx context.Context, client *spanner.Client, targetTables
 
 	truncateAll := true
 	targets := make(map[string]bool, len(targetTables))
-	if len(targetTables) > 0 {
+	whitelists := make(map[string]bool, len(excludeTables))
+	if len(targetTables) > 0 || len(excludeTables) > 0 {
```

Comment:
Does this actually work?

If this condition matches, `truncateAll = false` and `targetTables = []`, then no tables are truncated after all.

I tested this and it didn't work correctly.
```
$ ./spanner-truncate -p <PROJECT> -i <INSTANCE> -d <DATABASE> --exclude-tables=T3
Fetching table schema from ...

Rows in these tables will be deleted. Do you want to continue? [Y/n]
```
↑ It should have tables listed, but no tables were listed.